modules/libSceVrTracker/types.h

```diff
@@ -0,0 +1,161 @@
+#pragma once
+#include "codes.h"
+#include "modules_include/sceTypes.h"
+
+enum class SceVrTrackerProfile {
+  SCE_VR_TRACKER_PROFILE_000 = 0,
```

Comment:
You can remove SCE_VR_TRACKER_ from all the enums. The enum name already contains it.